schemas.py

```diff
@@ -0,0 +1,38 @@
+from datetime import date
+from typing import Optional
+
+from pydantic import BaseModel
+
+
+class AuthorBase(BaseModel):
+    name: str
+    bio: Optional[str] = None
+
+
+class AuthorCreate(AuthorBase):
+    pass
+
+
+class Author(AuthorBase):
+    id: int
+
+    class Config:
+        from_attributes = True
+
+
+class BookBase(BaseModel):
+    title: str
+    summary: str
```

Comment:
The `summary` field in the `BookBase` model is not marked as optional, but it should be optional to align with the database schema where `summary` is nullable. Consider changing it to `Optional[str] = None`.